src/Compilers/CSharp/Test/Semantic/Semantics/PropertyFieldKeywordTests.cs

```diff
@@ -261,8 +374,8 @@ public class MyAttribute : System.Attribute
             Assert.Equal(0, accessorBindingData.NumberOfPerformedAccessorBinding);
         }
 
-        [Fact]
-        public void TestFieldInLocalFunction_ShadowedByConstant()
+        [Theory, CombinatorialData]
+        public void TestFieldInLocalFunction_ShadowedByConstant([CombinatorialValues("always", "never")] string runNullableAnalysis)
```

Comment:
>[CombinatorialValues("always", "never")] string runNullableAnalysis [](http://example.com/codeflow?start=64&length=67)

I do not think this makes any difference unless we are utilizing SemanticModel binding. Are you going after something specific?

Reply:
Yes I don't know why I introduced this in the first place. Removed it